src/ceres/launchpad.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Launchpad | SORA Docs"
+head:
+  - - meta
+    - name: description
+      content: "Ceres Launchpad is the first station for new projects on the SORA network. "
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, launchpad, Polkaswap, SORA network"
+---
+
+# Ceres Launchpad
+
+The [Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 
+With Launchpad, new projects can easily launch their tokens on Polkaswap, and users have added security and peace of mind thanks to the Ceres Liquidity Locker, preventing pump and dump schemes.
+The Ceres [Liquidity Locker](/liquidity-locker.md) and [Token Locker](/token-locker.md) are integrated into the Launchpad.
```

Comment:
```suggestion
The Ceres [Liquidity Locker](./liquidity-locker.md) and [Token Locker](./token-locker.md) are integrated into the Launchpad.
```

Reply:
pls commit this urgent @DanijelCBS ! 
Cheers

Reply:
Fixed